sdk/textanalytics/ai-text-analytics/src/textAnalyticsClient.ts

```diff
@@ -354,7 +354,16 @@ export class TextAnalyticsClient {
         code: CanonicalCode.UNKNOWN,
         message: e.message
       });
-      throw e;
+      if (e.response.parsedBody.error.innererror.code === "InvalidDocumentBatch") {
+        const innerError = {
+          statusCode: e.statusCode,
+          code: e.response.parsedBody.error.innererror.code,
+          message: e.response.parsedBody.error.innererror.message
+        };
+        throw innerError;
```

Comment:
You need to turn this into a `RestError` the same way it was done in utils.ts before this diff:

```
throw new RestError(message, code, statusCode)
```

Reply:
done.